.github/workflows/ci.yml

```diff
@@ -332,6 +332,20 @@ jobs:
           - suite-tpcds
           - suite-oauth2
         exclude:
+          - config: config-cdh5
+            ignore exclusion if: >-
```

Comment:
> I think ignore-exclusion-marker is passable, but I prefer ignore exclusion if precisely because it looks like a special syntax

from syntax perspective yours is tons better than my proposal

what i am trying to avoid is having people search github workflows docs only to realize they cannot find the documentation for this syntax... which is what i did on first review pass

Reply:
Anyway, I am OK with current syntax as well

Reply:
I've updated the comment on the default value to be more explicit about how it works, so if anyone gets confused, they only have to look a little further down for the explanation.